cluster/juju/layers/kubernetes-worker/tests/00-setup

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I agre, thanks for catching this. We really need to update the charm template to do so